spirit/core/conf/defaults.py

```diff
@@ -12,7 +12,7 @@
 #: The category is auto-created and so this value should not change
 ST_TOPIC_PRIVATE_CATEGORY_PK = 1
 #: Enable/disable category ordering.
-ST_ORDERED_CATEGORIES = False
+ST_ORDERED_CATEGORIES = True
```

Comment:
change it to False. Use `@settings_override(ST_ORDERED_CATEGORIES=True)` in the test

Reply:
I couldn't use ```@override_settings``` as it doesn't work with url resolving. So I've made it the same way you deal with ```ST_UPLOAD_FILE_ENABLED```.